src/ja/worker/proxy/proxy.py

```diff
@@ -0,0 +1,40 @@
+from ja.common.proxy.proxy import ContinuousProxy
+from ja.common.message.server import ServerResponse
+from ja.common.proxy.ssh import SSHConfig
+
+
+class WorkerServerProxy(ContinuousProxy):
+    """
+    Proxy for the central server used on the worker client.
+    """
+    def __init__(self, ssh_config: SSHConfig):
+        """!
+        @param ssh_config: Config for paramiko
+        """
+
+    def register_self(self) -> ServerResponse:
+        """! Adds the worker client to the worker client pool of the central
+        server. The server will establish an SSHConnection to the worker client
```

Comment:
establish a connection to the server, not to the wc

Reply:
To send this command, the worked needs to have already set up the SSHConnection to the server.
This is so that the server then also creates an SSHConnection to the worker.

Reply:
I misread the comment, sorry.